src/libs/ReportActionsUtils.ts

```diff
@@ -494,12 +526,26 @@ function filterOutDeprecatedReportActions(reportActions: ReportActions | null):
  * to ensure they will always be displayed in the same order (in case multiple actions have the same timestamp).
  * This is all handled with getSortedReportActions() which is used by several other methods to keep the code DRY.
  */
-function getSortedReportActionsForDisplay(reportActions: ReportActions | null, shouldMarkTheFirstItemAsNewest = false): ReportAction[] {
-    const filteredReportActions = Object.entries(reportActions ?? {})
-        .filter(([key, reportAction]) => shouldReportActionBeVisible(reportAction, key))
-        .map((entry) => entry[1]);
+function getSortedReportActionsForDisplay(reportActions: ReportActions | null, shouldIncludeInvisibleActions = true): ReportAction[] {
+    let filteredReportActions;
+
+    if (shouldIncludeInvisibleActions) {
```

Comment:
This is backwards – if `shouldIncludeInvisibleActions` is true we filter invisible actions out, which is not right. This should be:

```suggestion
    if (!shouldIncludeInvisibleActions) {
```

Reply:
👍🏻